src/js/wrap.cpp

```diff
@@ -892,6 +943,27 @@ namespace js
           0));
     }
 
+    if (network_state != nullptr)
+    {
+      if (txctx == nullptr)
+      {
+        throw std::logic_error("Tx should be set to set network context");
+      }
+
+      auto network = JS_NewObjectClass(ctx, network_class_id);
+      JS_SetOpaque(network, network_state);
+      JS_SetPropertyStr(ctx, ccf, "network", network);
+      JS_SetPropertyStr(
+        ctx,
+        network,
+        "latestLedgerSecretSeqno",
```

Comment:
`getLatestLedgerSecretSeqno` ?

Reply:
I find it needlessly verbose, change my mind :)

Reply:
I think it would be more consistent with the rest of the exposed functions. With the name `latestLedgerSecretSeqno` I would expect a value property and not a function.

Reply:
Ok, fine.